### DISCUSSION TITLE
How long does it take to run the fuzzing test?

### DISCUSSION THREAD

**Discussion by epsilon-0** [uNetworking/uWebSockets#1334] [Q&A] [answered]
I've been able to build the fuzzing tests and am able to run them. They seem to be taking quite a while with lots of output, I'm not sure how long do they take to run fully.

Thanks!

**Comment by ghost** [comment, marked as answer]
They run forever ;)

**Reply by epsilon-0** [reply]
~~lol, I don't know if you are kidding or not, but it sure feels like that XD~~

I got more familiar with libfuzzer and it looks like it has some timeout options and other tinkerings. That sounds like what I need for now. Thanks!

**Reply by ghost** [reply]
They do run forever. You run them 24/7 that's why Google is offering to fuzz for open source projects.

**Reply by epsilon-0** [reply]
:O TIL!
Thanks!
